index.html

```diff
@@ -0,0 +1,600 @@
+      min-height: 252.99px;
+    }
+
+    .image>.height24 {
+      height: 24px;
+    }
+
+    .title {
+      font-size: 7vw;
+    }
+
+    .footer {
+      width: 100%;
+      background-color: #e7e7e7;
+    }
+
+    .paddingTop10 {
+      padding-top: 10vh;
+    }
+
+    .margin10 {
+      margin: 10vh;
+    }
+
+    .goldenButton {
+      background-color: #ffc107;
+    }
+
+    .height50 {
+      height: 50px;
+    }
+
+    .height20vh {
+      height: 20vh;
+    }
+
+    .marginTop20 {
+      margin-top: 20vh;
+    }
+
+    .marginTop5 {
+      margin-top: 5vh;
+    }
+
+    .marginTop10 {
+      margin-top: 10vh;
+    }
+
+    .circle {
+      width: 170px;
+      height: 170px;
+      border-radius: 50%;
+      padding: 5vh;
+    }
+
+    .circle2 {
+      width: 280px;
+      height: 280px;
+      border-radius: 50%;
+      padding: 10vh;
+    }
+
+    .move{
+      transform: translate(15vh, -5vh);
+    }
+
+    .move2{
+      transform: translate(-15vh, 5vh);
+    }
+  </style>
+</head>
+
+<body>
+  <!-- Navbar -->
+  <nav class="navbar fixed-top navbar-expand-lg navbar-light bg-light">
+    <a class="navbar-brand" href="/"><img class="height50" src="static/logo.png" /></a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+      aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav ml-auto p-2">
+        <li class="nav-item">
+          <a class="nav-link" href="/#about">About</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/#testimonials">Testimonials</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/#statistics">Statistics</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/#sponsors">Sponsors</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/#faq">FAQ</a>
+        </li>
+        <!-- <li class="nav-item">
+          <a class="nav-link btn" style="background-color: #ffc107" target="_blank" href="https://thegoldenhack.typeform.com/to/W8irwb">
+            Apply Now!
+          </a>
+        </li> -->
+      </ul>
+    </div>
+  </nav>
+
+  <!-- Header -->
+  <section class="container mt-5 mb-5 pt-5 pb-5">
+    <h1 class="text-center title">The<br />GoldenHack</h1>
+  </section>
+
+  <!-- About -->
+  <section id="about" class="container mt-5 mb-5 pt-5 pb-5">
+    <div class="row align-items-center">
+      <div class="col-md">
+        <img src="static/code.svg" class="image" />
+      </div>
+
+      <div class="col-md mx-1 my-5">
+        <h2 class="text-center mb-3"># What is The GoldenHack?</h2>
+        <p class="text-center">
+          <b>The GoldenHack</b> is a business hackathon held at Laurier on<br />

```

Comment:
I think it should be "... held at Laurier in October 2020."

Reply:
@tailaiwang 👀

Reply:
Fixed it in the latest commit!